gateway/crates/gateway-binary/src/main.rs

```diff
@@ -59,7 +62,15 @@ fn main() -> anyhow::Result<()> {
         let crate_version = crate_version!();
         tracing::info!(target: GRAFBASE_TARGET, "Grafbase Gateway {crate_version}");
 
-        federated_server::serve(args.listen_address(), config, args.fetch_method()?, otel_tracing).await?;
+        federated_server::serve(ServerConfig {
+            listen_addr: args.listen_address(),
+            config,
```

Comment:
:zap: **Suggestion** 
Ensure that the `config` variable is properly initialized and contains valid data before passing it to `federated_server::serve`. This will help avoid potential issues during runtime.